presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -169,6 +169,8 @@ Property Name                                      Description
                                                    See :ref:`hive-file-based-authorization` for details.
 
 ``hive.non-managed-table-writes-enabled``          Enable writes to non-managed (external) Hive tables.         ``false``
+
+``hive.create-non-managed-table-enabled``          Enable creating non-managed (external) Hive tables.          ``true``
```

Comment:
Maybe call it `hive.non-managed-table-create-enabled` to be consistent with `hive.non-managed-table-writes-enabled`?